src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -1027,8 +1027,8 @@ export default class DefaultDeviceController
     }
   }
 
-  private async listDevicesOfKind(deviceKind: string): Promise<MediaDeviceInfo[]> {
-    if (this.deviceInfoCache === null || !this.isWatchingForDeviceChanges()) {
+  private async listDevicesOfKind(deviceKind: string, force: boolean): Promise<MediaDeviceInfo[]> {
```

Comment:
nit: Could we be more specific what this parameter do? How about just clearCache?

Reply:
@ltrung It's more like updating than clearing the cache. We just run `updateDeviceInfoCacheFromBrowser` with parameter set to true. How about `forceUpdateCacheFromBrowser` or ` forceUpdateCache` or `forceUpdate`?

Reply:
forceUpdate sounds good to me.

Reply:
Updated.